Jenkinsfile: use env var to set owner/repo for github notifs

meowsbits opened the pull request:
This allows meowsbits (and other forks) to
receive proper build progress notifications.

Signed-off-by: meows <b5c6@protonmail.com>

tzdybal commented:
My configuration looks completely different:
```
{
    "networkId": 1,
    "chainId": 61,
    "daoForkBlock": 1920000,
    "eip150Block": 2500000,
    "eip155Block": 3000000,
    "eip160Block": 3000000,
    "petersburgBlock": 9573000,
    "ecip1010PauseBlock": 3000000,
    "ecip1010Length": 2000000,
    "ecip1017FBlock": 5000000,
    "ecip1017EraRounds": 5000000,
    "disposalBlock": 5900000,
    "ethash": {},
    "trustedCheckpoint": null,
    "trustedCheckpointOracle": null,
    "requireBlockHashes": null
}
```
It shows clearly that my db was created with "old school" way of defining forks.

meowsbits commented on cmd/echaindb/LICENSE:
```diff
@@ -0,0 +1,202 @@
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
I just `rm`ed the LICENSE.

meowsbits commented on cmd/echaindb/cmd/readChainconfig.go:
```diff
@@ -0,0 +1,77 @@
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
```
I just `rm`ed the LICENSE. https://github.com/etclabscore/multi-geth/pull/133/commits/2d7416fea26f8099ffd42c0435ece62153d192e5

meowsbits commented:
Removed LICENSE. Prints raw JSON from db now, instead of putting it through the `Unmarshal`er first.

meowsbits committed:
Jenkinsfile: use env var to set owner/repo for github notifs

This allows meowsbits (and other forks) to
receive proper build progress notifications.

Signed-off-by: meows <b5c6@protonmail.com>
